src/js/components/Data/propTypes.js

```diff
@@ -1,7 +1,11 @@
 import PropTypes from 'prop-types';
 
 const viewType = PropTypes.shape({
-  properties: PropTypes.shape({}),
+  properties: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.any),
+    PropTypes.shape({}),
+    PropTypes.any,
```

Comment:
Why is this variation needed? From the typescript typings, it appears that it's either an array of strings or an object.

Reply:
@ericsoderberghp, I check the type-definition and added the same.